js/map.js

```diff
@@ -45,25 +77,63 @@ map.on("load", () => {
     },
   })
 
-  map.addLayer({
-    id: "ihs-borders",
-    type: "line",
-    source: "ihs_rollup_source",
-    "source-layer": "ihs_rollup-2jzfoj",
-    paint: {
-      "line-color": "#000000",
-      "line-width": [
-        "case",
-        ["boolean", ["feature-state", "selected"], false],
-        5,
-        1,
-      ],
-    },
-  })
+  map
+    .addLayer({
+      id: "ihs-borders",
+      type: "line",
+      source: "ihs_rollup_source",
+      "source-layer": "ihs_rollup_categorized-bu6v14",
+      paint: {
+        "line-color": "#000000",
+        "line-width": [
+          "case",
+          ["boolean", ["feature-state", "selected"], false],
+          5,
+          1,
+        ],
+      },
+    })
+    .once("idle", () => {
+      // Once resources are loaded, set options for the searchable select input
```

Comment:
Big chunk added here, but this seemed like the best place to do this. In order to get the areas from mapbox, we need to wait for the app to receive the response. This `idle` event marks when the map isn't actively loading anything, and the `once()` method ensures that this only runs the first time it becomes idle.